recipes/tachyon/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set version = "0.99b6" %}
+{% set name = "tachyon" %}
+{% set md5 = "9e5ed4045c472e01e57480d7c7eee849" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://pkgs.fedoraproject.org/repo/pkgs/{{ name }}/{{ name }}-{{ version }}.tar.gz/{{ md5 }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
Why are we getting this from Fedora? Are there not sources anywhere? FWIW also had trouble finding the original sources too.